sections/loans/components/ActionBox/ActiveBorrowsTab/ModifyLoanActions/Draw.tsx

```diff
@@ -62,6 +84,7 @@ const Repay: React.FC<RepayProps> = ({ loan, loanId, loanTypeIsETH, loanContract
 						onTxConfirmed: () => {},
 					}),
 			});
+			router.push('/loans/list');
```

Comment:
dunno if the routes.ts is still maintained?

Reply:
Yea I think it is. Nice catch, should be using constants